src/chart/options/config.cpp

```diff
@@ -35,7 +35,8 @@ void Config::setParam(const std::string &path,
 		auto it = accessors.find(path);
 		if (it == accessors.end())
 			throw std::logic_error(
-			    "invalid config parameter: " + path);
+			    std::string(path) + "/" + value
```

Comment:
Here we should put path only once into the error message

Reply:
path is std::string, so conversion is unnecesarry. I unified all error message to the following format:

path[/value]: message